File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cs
public class Solution {
    public int[] TopKFrequent(int[] nums, int k) {
        Dictionary<int, int> count = new();
        int[] returnArray = new int[k];
        int counter = 0;

        foreach(int num in nums)
        {
            if (count.ContainsKey(num))
                count[num] += 1;
            else
                count.Add(num, 1);
        }

       
        foreach (var num in count.OrderByDescending(x => x.Value))
        {
            returnArray[counter] = num.Key;
            counter++;
            if (counter == k)
                break;
        }

        return returnArray;
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> newVector (nums.size() * 2);

        for(int i=0; i < nums.size(); i++) {
            newVector[i] = nums[i];
            newVector[nums.size()+i] = nums[i]; 
        }
        return newVector;
    }
};